builder/googlecompute/step_start_tunnel.go

```diff
@@ -307,7 +311,7 @@ func (s *StepStartTunnel) Run(ctx context.Context, state multistep.StateBag) mul
 		RetryDelay: (&retry.Backoff{InitialBackoff: 200 * time.Millisecond, MaxBackoff: 30 * time.Second, Multiplier: 2}).Linear,
 	}.Run(ctx, func(ctx context.Context) error {
 		// tunnel launcher/destroyer has to be different on windows vs. unix.
-		err := s.tunnelDriver.StartTunnel(ctx, tempScriptFileName)
+		err := s.tunnelDriver.StartTunnel(ctx, tempScriptFileName, s.IAPConf.IAPTunnelLaunchTimeout)
```

Comment:
ah, it would have been so nice if cancelling the ctx would simply kill that command !